src/Features/WcPayPromotion/WCPaymentGatewayPreInstallWCPayPromotion.php

```diff
@@ -24,9 +24,13 @@ class WCPaymentGatewayPreInstallWCPayPromotion extends \WC_Payment_Gateway {
 	 * Constructor
 	 */
 	public function __construct() {
+		$wc_pay_spec              = Init::get_wc_pay_promotion_spec();
 		$this->id                 = static::GATEWAY_ID;
-		$this->title              = __( 'WooCommerce Payments', 'woocommerce-admin' );
-		$this->method_description = ''; // will be replaced by wc.com data.
+		$this->method_title       = $wc_pay_spec->title;
```

Comment:
The title and the images in the promotion row (psuedo gateway) don't seem to change based on the REST API response, but subtitle and description works as expected.  Are they meant to be able to control those elements?

Reply:
Yeah the images should be coming from wc.com and rendered as a subtitle (@this->title actually - kinda confusing).
There should only be a very subtle change for the images as well, basically only removing a dash between the title and the images (sub title). When you inspect the images are they coming from wc.com and are they wrapped in img tags?

Reply:
No I saw the SVGs and not the image tags.  Which is odd because the description was in fact coming from my local WCCOM site.

Reply:
@joshuatf you might have to clear that transient again, it could be that it was using some old data still? (if you happen to have loaded this branch in the past) - or I broke something which could be very valid as well.